felix/fv/flowlogs/flow_tester.go

```diff
@@ -171,16 +172,6 @@ func (t *FlowTester) PopulateFromFlowLogs(reader FlowLogReader) error {
 		t.flows[fm] = fl
 	}
 
-	if t.options.CheckFlowsCompleted {
```

Comment:
I know we may not be using this anymore, but we can keep this in the tester's functionality, just in case we have a specific use case in the future